matrix/common/utils.py

```diff
@@ -0,0 +1,30 @@
+import pandas
```

Comment:
why is this in the PR? I donn't see any file named utils.py anymore. It should be pandas_utils.py. We are also planning on moving this function into the dss zarr store class?

Reply:
Looks like my changes got mixed up. Agreed this shouldn't be here. Removing from `develop`